### PR TITLE
LET-380 Add GMaps Places API key to FE infrastructure

### PR DESCRIPTION
**Pull request by tiagojsag**
⚠️ Before submitting the PR, make sure:

- You asked someone to review it
- You chose a meaningful title
- You ran the tests
    
Keep in mind tests are also automatically ran in every PR, and you should never merge a PR that has failed tests.

---

Description of the PR

## Testing instructions

What to test? How to do it?

## Tracking

Link to the task(s), if any

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+16.14
```
Comment (clementprdhomme) [inline .nvmrc:1]: There's a `.nvmrc` file in `frontend` already 😉.

**Review by clementprdhomme — APPROVED** [review]

**Comment by clementprdhomme** [comment]
Could you maybe update the `.env.sample` inside `frontend`?

File: .nvmrc (hunk 1 above)
Comment (tiagojsag) [reply .nvmrc:1]: there's a package.json in the frontend folder, and another one at the root.

**Comment by clementprdhomme** [comment]
Thank you @tiagojsag!

**Pull request merged**